Public/Src/Cache/ContentStore/App/CopyFile.cs

```diff
@@ -64,12 +65,6 @@ internal sealed partial class Application
                     {
                         _logger.Debug($"Copy of {hashString} to {finalPath} was successful");
                     }
-
-                    var shutdownResult = rpcClient.ShutdownAsync(context).Result;

```

Comment:
Just curious, does it mean that the code without this change, the old code was broken because we were shutting down pulled clients?

Reply:
If this is the case, then I would suggest adding an assertion in clientCache.CreateAsync to make sure the instance is opened and not shut down.

---
In reply to: [292213563](https://github.com/microsoft/BuildXL/pull/437#discussion_r292213563) [](ancestors = 292213563)